canonical-web-and-design/maas-ui
# feat(ui): update Usabilla to the 2.9 campaign

## Done

- Change the Usabilla code in Footer to the 2.9 campaign.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)

### QA steps

- Change `const debug = false;` in App.js.
- Load a react page and click on "Give feedback" in the footer. You should get a popup.
